pom.xml

```diff
@@ -13,20 +13,18 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <maven.surefire.version>2.22.2</maven.surefire.version>
-        <maven.failsafe.version>2.22.2</maven.failsafe.version>
         <junit.version>5.10.2</junit.version>
         <junit.platform.version>1.10.2</junit.platform.version>
-        <mockito.version>4.11.0</mockito.version>
+        <mockito.version>5.11.0</mockito.version>
         <jackson.version>2.17.0</jackson.version>
         <jackson.databind.version>2.17.0</jackson.databind.version>
         <spring.version>5.3.33</spring.version>
         <spring-security.version>5.8.11</spring-security.version>
-        <commons-io.version>2.15.1</commons-io.version>
+        <commons-io.version>2.16.1</commons-io.version>
         <immutables.version>2.10.1</immutables.version>
         <cloudfoundry-client.version>5.12.1.RELEASE</cloudfoundry-client.version>
-        <reactor-netty.version>1.0.39</reactor-netty.version>
-        <micrometer.version>1.12.4</micrometer.version>
+        <reactor-netty.version>1.1.17</reactor-netty.version>
```

Comment:
Did we test that oss cf-java-client still works and requests are tracked in dynatrace?

Reply:
yes requests are tracked in dynatrace